src/model_config_tests/__main__.py

```diff
@@ -0,0 +1,18 @@
+import os
+import sys
+
+# Running pytests using --pyargs does not run pytest_addoption in conftest.py
+# Using workaround as described here:
+# https://stackoverflow.com/questions/41270604/using-command-line-parameters-with-pytest-pyargs
+HERE = os.path.dirname(__file__)
+
+
+def main():
+    import pytest
```

Comment:
![image](https://github.com/ACCESS-NRI/model-config-tests/assets/45781416/f745745f-c616-41c1-94b2-82596bf07e58)
There are a whole bunch of these warnings [on the Files view](https://github.com/ACCESS-NRI/model-config-tests/pull/22/files) of this PR...what do they mean? Can it be turned off? :grin: Unless it's just because it's the initial run of the CodeCov stuff

Reply:
Looks like codecov goes over every line changed in the PR, and displays if it has been covered by tests.. I mean this project has a very low test coverage so far, so it's probably good to have codecov reminder to have more tests in future. They can be disabled in the codecov config, or if in the files view, you can press `a` to toggle the annotations on/off (https://docs.codecov.com/docs/github-checks#disabling-github-checks-patch-annotations)